src/views/endorse-company-closure-certificate.njk

```diff
@@ -30,7 +30,7 @@
       }}
       <h2 class="govuk-heading-m">Important notice for all interested parties</h2>
 
-      <p class="govuk-body">The company has applied to be struck off and dissolved.
+      <p id="paragraph-statement"class="govuk-body">The {% if approvalModel.officerType === 'member' %} LLP {% else %} company {% endif %} has applied to be struck off and dissolved.
```

Comment:
We are performing the same if else check too many times. This should be a filter i.e.
```
The {{.approvalModel.officerType | asCompanyDisplayName }} has applied to be struck off and dissolved.
```

This will require a new filter to be created

Reply:
Do L22 and L23 need to be updated too?